mirage/fixtures/registration-providers.ts

```diff
@@ -10,6 +10,7 @@ function randomAssets() {
 
 interface MirageRegistrationProvider extends RegistrationProvider {
     licensesAcceptableIds: string[];
+    brand: any;
```

Comment:
@aaxelb is this to be addressed in a later PR as well?

Reply:
It may be prudent to type this as `Partial<Brand>` in here

Reply:
Can it be null as well?

Reply:
since it's a type that's just used in this fixture (not exported), i'm ok letting this one `any` slip through for now... but yeah you're right, this should be addressed in that later PR

Reply:
Good call, I believe it can be `null` as well! And if it's only in this mirage fixture, it shouldn't be too big of a deal 👍